src/uct/ib/rc/accel/rc_mlx5_iface.c

```diff
@@ -664,19 +664,12 @@ uct_rc_mlx5_iface_qp_cleanup(uct_rc_iface_qp_cleanup_ctx_t *rc_cleanup_ctx)
 static ucs_status_t uct_rc_mlx5_iface_get_address(uct_iface_h tl_iface,
                                                   uct_iface_addr_t *iface_addr)
 {
-    uct_rc_mlx5_iface_common_t *iface    =
-            ucs_derived_of(tl_iface, uct_rc_mlx5_iface_common_t);
-    uct_ib_md_t *md                      = uct_ib_iface_md(&iface->super.super);
-    uct_rc_mlx5_iface_flush_addr_t *addr =
-            (uct_rc_mlx5_iface_flush_addr_t*)iface_addr;
-
-    addr->super.flags = UCT_RC_MLX5_TM_ENABLED(iface) ?
-                        UCT_RC_MLX5_IFACE_ADDR_FLAG_TM : 0;
+    uct_rc_mlx5_iface_common_t *iface = ucs_derived_of(tl_iface,
+                                                       uct_rc_mlx5_iface_common_t);
+    uct_rc_mlx5_iface_addr_t *addr    = (uct_rc_mlx5_iface_addr_t*)iface_addr;
```

Comment:
maybe remove `uct_rc_mlx5_iface_addr_t` type now, because this byte is not really flags, so `addr->flags` is confusing

Reply:
hmmm, ok, removed